### PR TITLE
The basic organize application app + simple admin

### PR DESCRIPTION
**Pull request by aniav**

### CODE REVIEW

**Comment by asendecka** [comment]
Looking at it now ❇️

File: organize/models.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django_date_extensions.fields import ApproximateDateField
+
+from core.models import Event
+from core.validators import validate_approximatedate
+
+involvement_choices = (
```
Comment (patrys) [inline organize/models.py]: This looks like a good candidate for `CONSTANT_CASE`

File: djangogirls/settings.py [hunk 2]
```diff
@@ -117,29 +117,54 @@
...
+         'url': 'https://organize.djangogirls.org/'},
+        {'label': 'Organizer\'s FAQ',
+         'icon': 'icon-bookmark',
+         'url': 'https://faq-organizers.djangogirls.org/'},
+        {'label': 'Organizer\'s Google Group',
+         'icon': 'icon-bookmark',
+         'url': 'https://groups.google.com/forum/#!forum/django-girls-organizers'},
+        {'label': 'Organizer\'s Slack',
+         'icon': 'icon-bookmark',
+         'url': 'https://djangogirls.slack.com/'},
```
Comment (patrys) [inline djangogirls/settings.py]: It's not clear what changed here because of all the cosmetic changes 😿

Comment (aniav) [inline djangogirls/settings.py]: Yeaah... I tend to do a lot of that. I have basically added the new app here. There should be a proper code cleanup in some other PR at some other time :)

**Comment by asendecka** [comment]
Looks awesome! 🐱

**Pull request merged**